### PR TITLE
Update csv filename

### PR DESCRIPTION
**Pull request by x-jokay**
Updates the csv filename.

### CODE REVIEW

File: docs/migrate/protect_card_locker/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ As Catima is based on Loyalty Card Keychain, importing your data from it is very
 ## 4. Press Export
 ![Step 4](step_4.png)
 
-## 5. Choose a save location, filename (default: LoyaltyCardKeychain.csv) and press Save
+## 5. Choose a save location, filename (default: Catima.csv) and press Save
```
Comment (TheLastProject) [inline docs/migrate/protect_card_locker/index.md:17]: This change is actually incorrect, take a look at the rendered HTML page on https://thelastproject.github.io/Catima/migrate/protect_card_locker/. Step 5 is still about getting your data out of Loyalty Card Keychain, which uses LoyaltyCardKeychain.csv as default filename.

Comment (x-jokay) [reply docs/migrate/protect_card_locker/index.md:17]: Well, then what about [this](https://github.com/TheLastProject/Catima/blob/master/app/src/main/java/protect/card_locker/ImportExportActivity.java#L76)?

Comment (TheLastProject) [reply docs/migrate/protect_card_locker/index.md:17]: Yes, that is the default name for Catima. But read the guide more closely. The guide is about exporting your data from Loyalty Card Keychain into Catima. Step 1 is opening Loyalty Card Keychain, everything up to and including 6 is in Loyalty Card Keychain. Step 7 is opening Catima and further steps are inside Catima.

Comment (x-jokay) [reply docs/migrate/protect_card_locker/index.md]: Reverted the change for this file.

**Pull request merged**